### PR TITLE
Rodeo IDE with app section

### PR DESCRIPTION
**Pull request by AlbertDeFusco**
Also includes Windows start menu options

### CODE REVIEW

File: recipes/rodeo/build.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+$PYTHON setup.py install
+
+if [ `uname` == Darwin ]
+then
+    cp $RECIPE_DIR/rodeo_mac.command $PREFIX/bin
+fi
+
+# Add more build steps here, if they are necessary.
```
Comment (msarahan) [inline recipes/rodeo/build.sh]: Please remove comments here

File: recipes/rodeo/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,89 @@
+package:
+  name: rodeo
+  version: "0.4.4"
+
+source:
+  fn: rodeo-0.4.4.tar.gz
+  url: https://pypi.python.org/packages/source/r/rodeo/rodeo-0.4.4.tar.gz
+  md5: cbd93b992b508eb6f0915214b3201661
+#  patches:
```
Comment (msarahan) [inline recipes/rodeo/meta.yaml]: Please remove comments from conda skeleton in this file

File: recipes/rodeo/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,89 @@
+package:
+  name: rodeo
+  version: "0.4.4"
```
Comment (msarahan) [inline recipes/rodeo/meta.yaml]: Please template version.  See other feedstocks for example.  Here's one: https://github.com/conda-forge/affine-feedstock/blob/843017788bed7d4b6ee6ae89f91eabfecb23a7ef/recipe/meta.yaml

**Comment by msarahan** [comment]
Travis failure is because you need to add readline to the Mac dependencies in meta.yaml: https://travis-ci.org/conda-forge/staged-recipes/jobs/130943126#L1334

**Comment by AlbertDeFusco** [comment]
@msarahan, should I be using `gnureadline` from pypi instead?

**Comment by msarahan** [comment]
If it works, sure.

**Comment by jakirkham** [comment]
This looks pretty slick. Thanks for adding this, @AlbertDeFusco.

**Comment by jakirkham** [comment]
Yeah, that seems to be what it is complaining about.

File: recipes/rodeo/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+app:
+  entry: rodeo .  # [not osx]
+  entry: open ${PREFIX}/bin/rodeo_mac.command  # [osx]
+  icon: logo.png
+  summary: Rodeo Data Science IDE
+  type: web
+
+about:
+  home: http://rodeo.yhat.com/
+  license: BSD
```
Comment (jakirkham) [inline recipes/rodeo/meta.yaml]: Please specify the number of clauses.

Comment (jakirkham) [inline recipes/rodeo/meta.yaml]: Also, can add `license_family: BSD` if you want.

**Comment by AlbertDeFusco** [comment]
Has `conda-forge` been added to the channels in Travis CI? Or is there now a way to specify the channel in the `requirements:` section?

**Comment by jakirkham** [comment]
> Has `conda-forge` been added to the channels in Travis CI?

Yes, we automatically add this.

**Comment by AlbertDeFusco** [comment]
@jakirkham, I've updated the license.

File: recipes/rodeo/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+    - jupyter
+    - flask >=0.10.1
+    - docopt
+    - pyzmq >=13
+    - mistune
+    - gnureadline  # [osx]
+
+  run:
+    - python
+    - jupyter
```
Comment (janschulz) [inline recipes/rodeo/meta.yaml]: Does that really need the full jupyter stack or just jupyter-client? Using jupyter is probably "saver" for users which do not add environments, but for users which use environments this pulls in the complete stack which I have installed in multiple environments...

File: recipes/rodeo/build.sh [hunk 6]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+$PYTHON setup.py install
```
Comment (jakirkham) [inline recipes/rodeo/build.sh]: Could we please add these arguments ( ` --single-version-externally-managed --record record.txt` )?

File: recipes/rodeo/bld.bat [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+set SCRIPT_DIR=%PREFIX%\Scripts
+mkdir %SCRIPT_DIR%
+
+copy %RECIPE_DIR%\rodeo.ico %MENU_DIR%
+if errorlevel 1 exit 1
+
+copy %RECIPE_DIR%\menu-windows.json %MENU_DIR%\rodeo.json
+if errorlevel 1 exit 1
+
+%PYTHON% setup.py install
```
Comment (jakirkham) [inline recipes/rodeo/bld.bat]: Could we please add these arguments ( ` --single-version-externally-managed --record record.txt` )?

**Comment by ocefpaf** [comment]
Ping @AlbertDeFusco there are a few minor comments above, but this looks almost ready to go. Can you take a quick look at the comments?

**Comment by jakirkham** [comment]
Friendly nudge @AlbertDeFusco. Any chance we can finish this one out?

File: recipes/rodeo/meta.yaml (hunk 5 above)
Comment (AlbertDeFusco) [inline recipes/rodeo/meta.yaml]: I tried using `jupyter_client` and rodeo just hung.

File: recipes/rodeo/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+    - flask >=0.10.1
+    - docopt
+    - pyzmq >=13
+    - mistune
+    - gnureadline  # [osx]
```
Comment (jakirkham) [inline recipes/rodeo/meta.yaml]: [Appears]( https://travis-ci.org/conda-forge/staged-recipes/jobs/142480808#L2167-L2175 ) `rodeo` requires `pip`. Could we please add it? Expect it should pass after that change.

File: recipes/rodeo/meta.yaml (hunk 5 above)
Comment (jakirkham) [inline recipes/rodeo/meta.yaml]: Hung where? On all OSes? Particular ones?

**Comment by jakirkham** [comment]
Any thoughts on this @AlbertDeFusco ?

**Comment by danielfrg** [comment]
Rodeo has been discontinued for a while since YHat was acquired, i imagine this can be closed.

**Pull request closed without merging**